src/fu-tool.c

```diff
@@ -1748,7 +1748,7 @@ fu_util_get_firmware_types (FuUtilPrivate *priv, gchar **values, GError **error)
 	g_autoptr(GPtrArray) firmware_types = NULL;
 
 	/* load engine */
-	if (!fu_engine_load (priv->engine, FU_ENGINE_LOAD_FLAG_NO_ENUMERATE, error))
+	if (!fu_engine_load_firmware (priv->engine, error))
```

Comment:
why not have a flag that `FU_ENGINE_LOAD_FIRMWARE_ONLY` you detect and keep the function the same?

Reply:
Alternate proposal: https://github.com/fwupd/fwupd/pull/2670